Header header
int64 current_speed
int64 current_angle

# Maximum sonar range
int64 max_sonar

# Maximum speed allowed
int64 max_speed

sensor_msgs/Image image